symphony.rb

```diff
@@ -0,0 +1,74 @@
+class Symphony < Formula
+  desc "Framework for solving mixed integer linear programs"
+  homepage "https://projects.coin-or.org/SYMPHONY"
+  url "http://www.coin-or.org/download/pkgsource/SYMPHONY/SYMPHONY-5.6.14.tgz"
+  sha256 "666d1c72c4bb084b470c4f066b1694f7b3b11a479de15d9d1e9bfe1b53e3ae63"
+  head "https://projects.coin-or.org/svn/SYMPHONY/trunk"
+
+  option "without-openmp", "Disable openmp support"
+  option "with-asl", "Build CLP with ASL support"
+  option "with-glpk", "Build with support for reading AMPL/GMPL models"
+  option "with-mumps", "Build CLP with MUMPS support"
+  option "with-openblas", "Build CLP with OpenBLAS support"
+  option "with-suite-sparse", "Build CLP with SuiteSparse support"
+
+  asl_dep = (build.with? "asl") ? ["with-asl"] : []
+  glpk_dep = (build.with? "glpk") ? ["with-glpk"] : []
+  mumps_dep = (build.with? "mumps") ? ["with-mumps"] : []
+  openblas_dep = (build.with? "openblas") ? ["with-openblas"] : []
+  suitesparse_dep = (build.with? "suite-sparse") ? ["with-suite-sparse"] : []
+
+  depends_on "homebrew/science/openblas" => :optional
+  depends_on "homebrew/science/glpk448" if build.with? "glpk"
+  depends_on "homebrew/science/asl" => :optional
+  depends_on "homebrew/science/mumps" => [:optional, "without-mpi"] + openblas_dep
+  depends_on "homebrew/science/suite-sparse" => [:optional] + openblas_dep
```

Comment:
I guess I haven't completely understood the dependencies mechanism yet. You are saying that I only need to list direct dependencies and not indirect, which I guess makes sense. I fixed that now.